plugins/source/github/client/client.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/cloudquery/plugin-sdk/v4/schema"
 	"github.com/gofri/go-github-ratelimit/github_ratelimit"
 	"github.com/google/go-github/v59/github"
+	"github.com/gregjones/httpcache"
+	"github.com/gregjones/httpcache/diskcache"
```

Comment:
Maybe we want to fork the repo if we are going to use it to ensure security vulnerabilities are flagged?

Reply:
We could, let me do a follow up commit